ideascube/serveradmin/management/commands/csv2pkg.py

```diff
@@ -13,8 +16,30 @@
 Optional columns are "lang", "preview", "kind", "tags".
 Attributes "path" and "preview" (if provided) must be paths
 (relative to the directory containing the CSV) to existing files.
+
+csv2pkg also generates a yaml to add to the catalog.
+Some attributes are automically calculated from the generated package.
+Other attributes may be provided as command arguments.
 """
 
+
+def generate_catalog_yaml_metadata(path, metadata, _type='zipped-medias'):
+    return {
+        'all' : {
+            metadata['package_id']: {
```

Comment:
These fields are displayed th their alphabetical order. Is it possible to fix an arbitrary order?
name, description, language, version, url, size, sha256sume, type.